sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/FilterConjunctionGroup.cs

```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#nullable disable
```

Comment:
I think we generally prefer to have nullable enabled. Why disable it here?

Reply:
I also prefer to have nullable enabled. :)
Adding .NET6 detected more places where nullables need to be considered.
Disabling here was the quick fix.
I think it would be better to address this as a follow up.

Reply:
I re-enabled the Nullables for the Filtering files. Please review my changes.